offset/qry.go

```diff
@@ -75,6 +76,10 @@ func (qry *offsetQuery) SeqWrap(wrap bool) error {
 	qry.seqWrap = wrap
 	return nil
 }
+func (qry *offsetQuery) Reverse(yes bool) error {
```

Comment:
Hmm, maybe better remove this? The way it is now looks like it's supported. Or, if we want it to still implement the interface, just return a "not implemented" error